group_vars/all.yml

```diff
@@ -14,20 +14,36 @@ default_node_types:
   - validator
   - default
 
-sui:
+tmp_dir: /tmp
+
+chainflip:
   validator:
```

Comment:
lets remove this validator and have type as node and engine, structure it like https://github.com/encapsulate-xyz/amplifier-ansible/blob/main/group_vars/all.yml#L17 please